<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>46</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>25</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="up">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../content/icons8-сортировать-во-возрастанию-50.png</normaloff>../content/icons8-сортировать-во-возрастанию-50.png</iconset>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="down">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../content/icons8-сортировать-по-убыванию-50.png</normaloff>../content/icons8-сортировать-по-убыванию-50.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tomat_name"/>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_minute">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_second">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
